libbeat/logp/config.go

```diff
@@ -24,10 +24,11 @@ import (
 // Config contains the configuration options for the logger. To create a Config
 // from a common.Config use logp/config.Build.
 type Config struct {
-	Beat      string   `config:",ignore"`   // Name of the Beat (for default file name).
-	JSON      bool     `config:"json"`      // Write logs as JSON.
-	Level     Level    `config:"level"`     // Logging level (error, warning, info, debug).
-	Selectors []string `config:"selectors"` // Selectors for debug level logging.
+	Beat       string   `config:",ignore"`   // Name of the Beat (for default file name).
+	JSON       bool     `config:"json"`      // Write logs as JSON.
+	Level      Level    `config:"level"`     // Logging level (error, warning, info, debug).
+	Selectors  []string `config:"selectors"` // Selectors for debug level logging.
+	ECSEnabled bool     `config:"ecs"`
```

Comment:
we want to enable this setting by default?

Reply:
I would consider changing the default a breaking change, as it changes the log format.

Reply:
Do you mind adding a comment to describe what this option does?

Reply:
good point - added a comment